package.json

```diff
@@ -73,11 +73,13 @@
     "react-transform-catch-errors": "1.0.0",
     "react-transform-hmr": "1.0.1",
     "redbox-react": "1.2.0",
+    "redux": "3.3.1",
     "rimraf": "2.5.1",
     "secure-handlebars": "1.2.0",
     "serialize-javascript": "1.1.2",
     "sha1": "1.1.1",
     "sinon": "1.17.2",
+    "temp": "0.8.3",
```

Comment:
Does this qualify as a `devDependency`? is it only used for tests? if so we can move it to `devDependencies` and in the future if you want that to happen automatically `npm install _insert_name_ --save-dev`